mooncake-store/src/master.cpp

```diff
@@ -67,6 +67,9 @@ DEFINE_int64(client_ttl, mooncake::DEFAULT_CLIENT_LIVE_TTL_SEC,
 DEFINE_string(cluster_id, mooncake::DEFAULT_CLUSTER_ID,
               "Cluster ID for the master service, used for kvcache persistence in HA mode");
 
+DEFINE_string(memory_allocator, "cachelib",
```

Comment:
Let's switch default to offset. The bug in cachelib hasn't been fixed yet.

Reply:
Sounds good. Changed to offset